src/pe/header.rs

```diff
@@ -30,8 +31,6 @@ impl DosHeader {
 #[repr(C)]
 #[derive(Debug, PartialEq, Copy, Clone, Default)]
 pub struct CoffHeader {
-    /// COFF Magic: PE\0\0, little endian
-    pub signature: u32,
```

Comment:
How come this changed? Or put another way, how does this work now, or how did it work before ?
Is the data layout of coff/PE such that it just works ?
Also this is technically breaking so we should add a breaking changes note, though I doubt anyone used it.

**edit** looking at this now looks like this is the second element of the header just before the coff header which means they’d have the same layout if it’s packed. Wow good fix/find :)

Reply:
Yeah I'm just moving this field around. I had to do this because COFF object files don't have any of the PE headers, including this magic value.